agent/lib/tasks/release.rake

```diff
@@ -11,7 +11,7 @@ namespace :release do
   end
 
   desc 'Build all'
-  task :build => [:build_docker, :build_ubuntu] do
+  task :build => [:build_docker, :build_ubuntu, :build_ubuntu_xenial] do
```

Comment:
I think it would make sense to have `build_ubuntu` task that runs `build_ubuntu_trusty` & `build_ubuntu_xenial` .. now it sounds a little bit strange?